internal/xdscache/v3/listener.go

```diff
@@ -288,14 +288,24 @@ type ListenerCache struct {
 }
 
 // NewListenerCache returns an instance of a ListenerCache
-func NewListenerCache(config ListenerConfig, address string, port int) *ListenerCache {
+func NewListenerCache(config ListenerConfig, address string, port int, enableDebugListener bool) *ListenerCache {
 	stats := envoy_v3.StatsListener(address, port)
-	return &ListenerCache{
+	admin := envoy_v3.AdminListener("127.0.0.1", 9001)
```

Comment:
the admin port is configurable currently and I think that flag has kinda been dropped in a sense?

should we deprecate the flag? wire it through to make the listen port here for the read-only admin page configurable?

Reply:
We could reuse the port flag, but maybe that's confusing? 

I could reuse it for now, introduce a new flag and deprecate the old.

Reply:
Ahh I see, I'm mixing up the bits. This would be a new flag on Contour to specify. Yeah let's do that, introduce a new config file item I guess and deprecate the bootstrap port.